src/mongocxx/test/database.cpp

```diff
@@ -38,6 +38,21 @@ TEST_CASE("A default constructed database is false-ish", "[database]") {
     REQUIRE(!d);
 }
 
+TEST_CASE("database copy", "[database]") {
+    client mongodb_client{uri{}};
+
+    std::string dbname{"foo"};
+    std::string dbname2{"bar"};
+    database db = mongodb_client[dbname];
+
+    database db2 = db;
+    database db3 = mongodb_client[dbname2];
+    db3 = db;
+
```

Comment:
I thought that db2 was testing the copy constructor, since db2 had not been previously assigned. Is that not how it works? Would "database db2 {db}" or "database db2 = database(db)" test the copy constructor?